### PR TITLE
Add navigation capabilities to the code.

### PR DESCRIPTION
**Pull request by lishan-fernando**
Add navigation capabilities to the code.
Adding code to the git repo for the first time.

### CODE REVIEW

**Comment by lishan-fernando** [comment]
@shreesotam - Please review

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 # IAS0060MU
 IAS0060 Robotics Project - μ-CAT
+Developers
```
Comment (shreesotam) [inline README.md]: please add email addresses of all the developers.

Comment (lishan-fernando) [reply README.md]: @shreesotam -  emaill addresses are added to the file. please have a look again.

Comment (shreesotam) [reply README.md]: ok

**Pull request merged**